app/assets/javascripts/discourse/app/components/d-toggle-switch.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="d-toggle-switch">
+  <label class="d-toggle-switch--label">
+    {{! template-lint-disable no-unnecessary-concat  }}
```

Comment:
This is necessary because writing it like:
`aria-checked={{@state}}` results in the DOM showing:

```
<button class="d-toggle-switch__checkbox" role="switch" aria-checked type="button"></button>
```

while, adding the quotes and writing it like `aria-checked="{{@state}}" results in the `true/false` appearing in the DOM like so:

```
<button class="d-toggle-switch__checkbox" role="switch" aria-checked="false" type="button"></button>
```